src/ontology/cob.Makefile

```diff
@@ -241,3 +248,60 @@ products/demo-cob.owl: $(TMPDIR)/demo-cob-merged.owl | products/
 
 $(TMPDIR)/subset-%.owl: $(TMPDIR)/merged-%.owl subsets/terms_%.txt | $(TMPDIR)
 	$(ROBOT) extract -m BOT -i $< -T subsets/terms_$*.txt -o $@
+
+# Crosscheck against upstream ontologies
+
+X_IMPORTS := BFO CHEBI CL DRON ENVO FOODON GO IAO MOP NCBITaxon OBI PATO PCO PO PR RO SO UBERON VO
+X_IMPORT_OWL_FILES := $(foreach n,$(X_IMPORTS), $(IMPORTDIR)/$(n)_import.owl)
+
+$(IMPORTDIR)/%_ontofox.txt: cob-edit.tsv
```

Comment:
Why use ontofox rather than ROBOT? Doesnt that add an unnecessary tool dependency? What can ontofox do that ROBOT cant?

Reply:
Because I need 1 term from NCBITaxonomy (1.5GB storage on disk, 16+ GB RAM, 10+ minutes to process), 1 term from Protein Ontology, 1 term from Drug Ontology, etc. I just need some annotations, nothing fancy. OntoFox can do all of this in 20 seconds.

Reply:
And if OntoFox is down, we have the copies in version control (see the next comment).

Reply:
Hm makes sense. Eventually this would be a great service to deploy for GUOBO as well: a single place to extract modules from all members of the grand unified OBO ontology!